graphql-dgs/src/test/kotlin/com/netflix/graphql/dgs/DgsSchemaProviderTest.kt

```diff
@@ -79,6 +79,36 @@ internal class DgsSchemaProviderTest {
         assertEquals("schema1.graphqls", findSchemaFiles[0].filename)
     }
 
+    @Test
+    fun findMultipleSchemaFilesSingleLocation() {
+        val findSchemaFiles = DgsSchemaProvider(
+            applicationContextMock,
+            Optional.empty(),
+            Optional.empty(),
+            Optional.empty(),
+            listOf("classpath*:location1/**/*.graphql*")
+        ).findSchemaFiles()
+        assertThat(findSchemaFiles.size).isGreaterThan(2)
+        assertEquals("location1-schema1.graphqls", findSchemaFiles[0].filename)
+        assertEquals("location1-schema2.graphqls", findSchemaFiles[1].filename)
+    }
+
+    @Test
+    fun findMultipleSchemaFilesMultipleLocations() {
+        val findSchemaFiles = DgsSchemaProvider(
+            applicationContextMock,
+            Optional.empty(),
+            Optional.empty(),
+            Optional.empty(),
+            listOf("classpath*:location1/**/*.graphql*", "classpath*:location2/**/*.graphql*")
+        ).findSchemaFiles()
+        assertThat(findSchemaFiles.size).isGreaterThan(4)
```

Comment:
Nice tests. Thx for adding @marceloverdijk . 

This ordered access happens to work because of naming of the files and that is totally fine and intentional. If that is not the case though I will reach out for some AssertJ fun such as (you probably already know about this, but just in case you don't): 

```java
        assertThat(findSchemaFiles)
                .extracting(Resource::getFilename)
                .containsAll(listOf(
                        tuple("location1-schema1.graphqls"),
                        tuple("location1-schema2.graphqls"),
                        tuple("location2-schema1.graphqls"),
                        tuple("location2-schema1.graphqls")))
```
I think its cleaner the way you have it written currently though.